http2_general/test_h2_frame.py

```diff
@@ -173,21 +173,7 @@ def test_rst_frame_in_response(self):
         client.stream_id = stream_with_rst
         client.send_request((self.get_request + [("host", "")], "asd"), "400")
 
-        # client open new stream
-        client.make_request(self.get_request, end_stream=True)
-        client.wait_for_response(3)
-
-        # client send DATA frame in stream 1 and it must be open.
-        client.stream_id = 1
-        client.make_request("body", end_stream=True)
-        client.wait_for_response(3)
```

Comment:
This part of test check status of streams. First stream must not be closed.

Reply:
Fixed